packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -74,8 +74,8 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
 
     return (
       <div className={ css('ms-Breadcrumb', className, styles.root) } ref='renderingArea'>
-        <FocusZone direction={ FocusZoneDirection.horizontal }>
-          <ul className={ css('ms-Breadcrumb-list', styles.list) }>
+        <FocusZone direction={ FocusZoneDirection.horizontal } role='navigation'>
```

Comment:
Can we have multiple navigation landmarks? I thought I'd remember that you should have only one, and then I'm not sure whether the breadcrumbs should be that landmark. Could be wrong though